packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address.jsx

```diff
@@ -42,4 +53,12 @@ export default connect(({ client, notifications, common }) => ({
     refreshNotifications: notifications.refreshNotifications,
     has_restricted_mt5_account: client.has_restricted_mt5_account,
     app_routing_history: common.app_routing_history,
+    account_settings: client.account_settings,
+    is_eu: client.is_eu,
+    addNotificationByKey: notifications.addNotificationMessageByKey,
```

Comment:
Shall we name the variables consistently everywhere? `addNotificationMessageByKey` instead of `addNotificationByKey`?
It becomes easier for the folks working on migrating from `connect` to `useStore`. One PR was reverted due to a mismatch like this.

Reply:
let me change it to follow the name 👌